packages/ketcher-core/src/infrastructure/services/struct/RemoteStructService.ts

```diff
@@ -77,12 +77,12 @@ function indigoCall(
   }
 }
 
-class IndigoService implements StructService {
-  private baseUrl: string
+class RemoteStructService implements StructService {
+  private apiPath: string
   private defaultOptions: any
 
-  constructor(base: string, defaultOptions: any) {
-    this.baseUrl = !base || /\/$/.test(base) ? base : base + '/'
+  constructor(apiPath: string, defaultOptions: any) {
+    this.apiPath = apiPath
     this.defaultOptions = defaultOptions
   }
```

Comment:
I'd prefer to add readonly for this fields:
```
  constructor(
    private readonly apiPath: string,
    private readonly defaultOptions: any
  ) {
  }
```

Reply:
agreed